### ISSUE TITLE
这个项目太棒了，可以实现视频压缩吗？

### ISSUE THREAD

**Issue by sxlvalue** [Martin20150405/Omoshiroi#34]
这是一个非常棒的项目，就是录了之后的视频有点稍大，要是能压缩就好了。

**Comment by Martin20150405** [comment]
目前可以在MediaVideoEncoder里面设置(以后应该会改掉)
>>
	private int calcBitRate() {
		final int bitrate = (int)(BPP * FRAME_RATE * mWidth * mHeight);
		Log.i(TAG, String.format("bitrate=%5.2f[Mbps]", bitrate / 1024f / 1024f));
		return bitrate;
	}

修改BPP即可，目前大约是每秒500k的样子（1080P的情况下），BPP和体积基本成正比

**Comment by snetts** [comment]
I must agree, it is a great project I should learn Chinese but at the moment I will use Google Translate to get what you are saying. In the README.md you mention that we should get a license from SenseTime if we want to use Face Detection. I have searched and failed to find where to even start the process of getting the license.

Secondly I was testing the demo app on Genymotion with ARM Translation installed but I am getting an error which was expected since that was a Genymotion bug. Quick question what are the alternatives for testing the project under a windows environment?

Finally the native code could you please make them in Intel format so that it is easier to test on Intel devices. Sorry for the eBook, but any help will be appreciated.
